packages/samples/atomic-react/project.json

```diff
@@ -1,3 +1,22 @@
 {
-  "name": "atomic-react-samples"
+  "name": "atomic-react-samples",
+  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+  "namedInputs": {
+    "negativeBuildOutputs": ["!{projectRoot}/public/dist"]
+  },
+  "targets": {
+    "cached:build": {
+      "outputs": ["{projectRoot}/public/dist"],
+      "executor": "nx:run-commands",
+      "options": {
+        "commands": ["tsc --noEmit", "tsc --module commonjs --noEmit"],
+        "parallel": true,
+        "cwd": "packages/samples/atomic-react"
+      }
+    },
+    "build": {
+      "dependsOn": ["cached:build"],
```

Comment:
Can we mark atomic build as a dep here ?

Reply:
There's no need. This task depends on the `atomic-react:cached:build` task, which itself depends on the build task of all dependencies (`^build`).